### PR TITLE
Update 02-project-intro.Rmd

### PR DESCRIPTION
**Pull request by raynamharris**
convirtió todo el código .md a .Rmd

### CODE REVIEW

**Comment by raynamharris** [comment]
@fmichonneau @orchid00 this is my first attempt to test drive the conversion. How do I know if it will render properly (other than by confirming that indeed all the text appears correct)?

File: _episodes_rmd/02-project-intro.Rmd [hunk 1]
```diff
@@ -14,6 +14,9 @@ keypoints:
 source: Rmd
 ---
 
+```{r, include=FALSE}
+source("../bin/chunk-options.R")
+knitr_fig_path("02-")
```
Comment (fmichonneau) [inline _episodes_rmd/02-project-intro.Rmd]: don't forget to close this chunk

**Comment by raynamharris** [comment]
Thanks, @fmichonneau! I noticed that my formatting was a little different but I didn't occur to me that I had forgotten to close a chunk. Now the rendering looks good.

File: _episodes_rmd/02-project-intro.Rmd (hunk 1 above)
Comment (raynamharris) [reply _episodes_rmd/02-project-intro.Rmd]: closed

**Comment by fmichonneau** [comment]
besides closing the first chunk, it looks good.

Now that the repository is structured as a standard lesson, you can use the make commands.

```sh
make lesson-md
```

can be used to check that the Rmd files can be converted to md without any issue.

```sh
make serve
```

(if you have Jekyll working) will build and make the website available in your web browser as it would if it was served by GitHub. It will be available at "localhost:4000" in your browser. If you don't have jekyll working, you can do

```sh
make site
``` 

and the content of the site will be available in the `_site` folder.

**Pull request merged**